CMSgov/easi-app
# EASI-2509 EASI-2547 Close & Re-open TRB Request Action (Admin)

# EASI-2509 EASI-2547
Close: https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=1779-107225&t=mM9AV1fuPIC5ia99-0
Reopen: https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=1779-107386&t=8m6RsnvvpIEzcIjL-0

## Changes and Description

`/trb/:id/:activePage/(close|reopen)-request` -> `<CloseRequest>`



## QA

- Start a trb request form
- Note the trb request uuid
- Complete the form at /trb/:id/initial-request-form/close-request
- Check that it redirects to request home
- Repeat the same steps for /trb/:id/initial-request-form/reopen-request, just make sure it's closed it first
